src/app/vendorEngramsXyzApi/vendorDrops.ts

```diff
@@ -2,8 +2,8 @@ export interface VendorDrop {
   id: number;
   vendor: VendorEngramVendor;
   type: VendorDropType;
-  verified: boolean;
-  enabled: boolean;
+  verified: number;
```

Comment:
These return 0 and 1 (for false and true); you just want to hew closer to the data we're returning, I guess?

Reply:
Yup, they won't really get converted to true/false just by the interface definition, so I'd like to model them as they are, since they will then require special handling (see above).